src/vue/common/TopBar.vue

```diff
@@ -43,6 +43,7 @@ export default {
   display: flex;
   align-items: center;
   min-height: 6.5rem;
+  max-height: 6.5rem;
```

Comment:
by this change you produced another bug (see https://app.asana.com/0/0/1107985761889108/f)

Reply:
ok